src/com/openattestation/4.0/alpha-schema.json

```diff
@@ -0,0 +1,205 @@
+        },
+        "type": { "$ref": "#/definitions/type" },
+        "name": {
+          "type": "string",
+          "description": "Issuer's name"
+        },
+        "identityProof": {
+          "type": "object",
+          "properties": {
+            "identityProofType": {
+              "type": "string",
+              "enum": ["DNS-TXT", "DNS-DID", "DID"]
+            },
+            "identifier": {
+              "type": "string",
+              "description": "Identifier to be shown to end user upon verifying the identity"
+            }
+          },
+          "required": ["identityProofType", "identifier"],
+          "additionalProperties": false
+        }
+      },
+      "required": ["id", "name", "identityProof"]
+    },
+    "credentialStatus": {
+      "oneOf": [
+        {
+          "type": "object",
+          "properties": {
+            "type": {
+              "$ref": "#/definitions/type"
+            },
+            "credentialStatusType": {
+              "type": "string",
+              "enum": ["NONE"]
+            }
+          },
+          "required": ["credentialStatusType"]
+        },
+        {
+          "type": "object",
+          "properties": {
+            "type": {
+              "$ref": "#/definitions/type"
+            },
+            "credentialStatusType": {
+              "type": "string",
+              "enum": ["REVOCATION_STORE", "OCSP_RESPONDER"]
+            },
+            "location": {
+              "type": "string",
+              "format": "uri",
+              "examples": ["https://ocsp-responder.example.com"]
+            }
+          },
+          "required": ["credentialStatusType", "location"]
+        }
+      ]
+    },
+    "renderMethod": {
+      "type": "object",
+      "properties": {
+        "type": { "$ref": "#/definitions/type" },
+        "renderMethodType": {
+          "type": "string",
+          "enum": ["EMBEDDED_RENDERER"],
+          "description": "Type of renderer template"
+        },
+        "name": {
+          "type": "string",
+          "description": "Template name to be use by template renderer to determine the template to use"
+        },
+        "url": {
+          "type": "string",
+          "format": "uri",
+          "pattern": "^(https?)://",
+          "description": "URL of a decentralised renderer to render this document"
+        }
+      },
+      "required": ["renderMethodType", "name", "url"]
+    },
+    "credentialSubject": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "uri",
+          "description": "URI when dereferenced, results in a document containing machine-readable information about the issuer that can be used to verify the information expressed in the credential. More information in https://www.w3.org/TR/vc-data-model/#issuer"
+        },
```

Comment:
the description might not be correct, i thought its the id of the object of claim
https://www.w3.org/TR/vc-data-model-2.0/#credential-subject

Reply:
Oh yes, good catch. Looking again at the [documentation](https://www.w3.org/TR/vc-data-model/#credential-subject), we can reword it to:

> An identifier related to the subject of the verifiable credential. More information in https://www.w3.org/TR/vc-data-model/#credential-subject